examples/pwa/pwa.js

```diff
@@ -17,7 +17,7 @@
 'use strict';
 
 function log(args) {
-  var var_args = Array.prototype.slice.call(arguments, 0);
+  const var_args = Array.prototype.slice.call(arguments, 0);
```

Comment:
I don't believe `pwa.js` is currently compiled. So we have to support a wider generation of JS here, unfortunately. I think `const` is still not supported in bunch of browsers.